sdk/maps/azure-maps-timezone/src/main/java/com/azure/maps/timezone/models/TimezoneIdOptions.java

```diff
@@ -0,0 +1,144 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+// See License.txt in the project root for license information.
+
+package com.azure.maps.timezone.models;
+
+import java.time.OffsetDateTime;
+/**
+ * Organizes inputs for get timezone by ID
+ */
+public final class TimezoneIdOptions {
+    private String timezoneId;
+    private String acceptLanguage;
+    private TimezoneOptions options;
+    private OffsetDateTime timeStamp;
+    private OffsetDateTime daylightSavingsTimeFrom;
+    private Integer daylightSavingsTimeLastingYears;
+
+    /**
+     * TimezoneIdOptions constructor
+     * @param timezoneId the IANA time zone ID.
+     */
+    public TimezoneIdOptions(String timezoneId) {
+        this.timezoneId = timezoneId;
+    }
+
+    /**
+     * get timezoneId
+     * @return the timezone id, the IANA time zone ID.
+     */
+    public String getTimezoneId() {
+        return timezoneId;
+    }
+
+    /**
+     * Set timezone id
+     * @param timezoneId the IANA time zone ID.
+     * @return TimezoneIdOptions
+     */
+    public TimezoneIdOptions setTimezoneId(String timezoneId) {
+        this.timezoneId = timezoneId;
+        return this;
+    }
+
+    /**
+     * get accept language
+     * @return the accept language. Specifies the language code in which the timezone names should be returned. If no language
+     *     code is provided, the response will be in "EN". Please refer to [Supported
+     *     Languages](https://docs.microsoft.com/azure/azure-maps/supported-languages) for details.
+     */
+    public String getAcceptLanguage() {
```

Comment:
we have `TimezoneCoordinateOptions.getLanguage()`, please use consistent API names

```suggestion
    public String getLanguage() {
```